dashboard/app/models/blockly.rb

```diff
@@ -294,4 +304,12 @@ def autoplay_blocked_by_level?
     # Wrapped since we store our serialized booleans as strings.
     self.never_autoplay_video == 'true'
   end
+
+  # Returns an array of all the contained levels
+  # (based on the contained_level_names property)
+  def contained_levels
+    names = properties["contained_level_names"]
+    return [] unless names.present?
+    Level.where(name: properties["contained_level_names"])
```

Comment:
Yes, I would like to fix this, but I think we should do it for LevelGroup, Contained Levels, and Project Template Levels all at the same time. I think the seed process will need to be changed to handle all of these in a different manner (and perhaps level builder to provide an early warning?)